dev/ts/pages/diagram-navigation.ts

```diff
@@ -26,37 +26,37 @@ function configureFitAndZoomButtons(): void {
 }
 
 function configureZoomThrottleInput(parameters: URLSearchParams): HTMLInputElement {
-  const elZoomThrottle = document.getElementById('zoom-throttle') as HTMLInputElement;
+  const elementZoomThrottle = document.getElementById('zoom-throttle') as HTMLInputElement;
   if (parameters.get('zoomThrottle')) {
-    elZoomThrottle.value = parameters.get('zoomThrottle');
+    elementZoomThrottle.value = parameters.get('zoomThrottle');
   }
-  return elZoomThrottle;
+  return elementZoomThrottle;
 }
 
 function configureZoomDebounceInput(parameters: URLSearchParams): HTMLInputElement {
-  const elZoomDebounce = document.getElementById('zoom-debounce') as HTMLInputElement;
+  const elementZoomDebounce = document.getElementById('zoom-debounce') as HTMLInputElement;
```

Comment:
**suggestion:** rename into `zoomDebounceElement` for consistency with the type and the common usage.